quay-enterprise/quay-ssl.md

```diff
@@ -0,0 +1,136 @@
+# SSL + Quay Enterprise
+
+**NOTE**: This document assumes you have deployed [Quay Enterprise as a single container.](https://tectonic.com/quay-enterprise/docs/latest/initial-setup.html)
```

Comment:
`This document assumes [Quay Enterprise has been deployed as a single container](https://tectonic.com/quay-enterprise/docs/latest/initial-setup.html).`